packages/adapter-vercel/index.js

```diff
@@ -82,7 +82,11 @@ const redirects = {
 };
 
 /** @type {import('.').default} **/
-export default function ({ external = [], edge, split } = {}) {
+export default function ({
```

Comment:
don't need to duplicate the types here. just declare the function as a `const` and then export it later